org_status/encoders/gitman.py

```diff
@@ -1,19 +1,46 @@
 import yaml
 from giturlparse import parse
+from gitman.git import clone
+import os
+import subprocess
+import tempfile
+import datetime
```

Comment:
Imports should be sorted. See: https://www.python.org/dev/peps/pep-0008/#imports